macros/create_hidden_shares_filter.yml

```diff
@@ -0,0 +1,3 @@
+definition: search *
```

Comment:
We dont neeed this macro file as its not being used by the detection search. Users should probably use the create_or_delete_windows_shares_filter macros to whitelist/blacklist